libqtile/config.py

```diff
@@ -630,8 +619,20 @@ def _comparator(_type, rule, value):
                 }
 
         for _type, rule in self._rules:
-            if _comparator(_type, rule, prop_getters[_type](client)):
```

Comment:
I'm not sure which version I prefer... (with or without the comparator function)